src/templates/list-table-block/index.vue

```diff
@@ -343,8 +343,11 @@
 
   const menuRef = ref({})
   const toggleActionsMenu = (event, selectedID) => {
+    if (!selectedID) {
+      throw new Error('rowData id not found')
```

Comment:
rowData id not found não fornece de imediato o que deve ser feito. Sugestão por trocar a mensagem para algo mais claro.
'Please provide an id for each data item through the service mapping'